test/unit/cli/opts.test.js

```diff
@@ -199,4 +167,29 @@ test('Options', async t => {
 
 function argv(... args) {
   return ['node', 'ez-build', ... args]
+}
+
+function generateCombinations(flags) {
```

Comment:
All possible non empty sets of parameters, each one of them in every possible order?